<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="status" name="header" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>36</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="mainStack">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>36</y>
      <width>800</width>
      <height>340</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(18, 22, 31);
background-color: rgb(18, 22, 31);</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="chart">
     <property name="styleSheet">
      <string notr="true">color: rgb(52, 101, 164);</string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>341</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="chartLayout">
       <property name="spacing">
        <number>7</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="config">
     <widget class="QProgressBar" name="ramUsage">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>80</y>
        <width>481</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar {
border: 1px solid black;
text-align: top;
	background-color: rgb(238, 238, 236);
width: 15px;
padding: 1px;

width: 15px;
}

QProgressBar::chunk {
background-color: rgb(114, 159, 207);

border: 1px solid black;
}
</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>50</y>
        <width>124</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(52, 101, 164);</string>
      </property>
      <property name="text">
       <string>RAM Usage</string>
      </property>
     </widget>
     <widget class="QLabel" name="IP">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>190</y>
        <width>521</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(238, 238, 236);</string>
      </property>
      <property name="text">
       <string>IP</string>
      </property>
     </widget>
     <widget class="QLabel" name="IP_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>190</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(238, 238, 236);</string>
      </property>
      <property name="text">
       <string>IP:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="dial_menu">
     <widget class="QDial" name="dial_config">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>90</y>
        <width>191</width>
        <height>181</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
    background-color: #27272B;
    color: #FFFFFF;
    qproperty-knobRadius: 5;
    qproperty-knobMargin: 5;

</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>100</y>
        <width>191</width>
        <height>161</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>60</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(238, 238, 236);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="led">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>139</y>
        <width>91</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(211, 215, 207);
background-color: rgb(18, 22, 31);
border-width:0px;

</string>
      </property>
      <property name="text">
       <string>led</string>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>21</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="navigation" name="bottom" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>376</y>
      <width>800</width>
      <height>104</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
   </widget>
   <widget class="QDial" name="dial">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>280</y>
      <width>191</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
    background-color: #27272B;
    color: #FFFFFF;
    qproperty-knobRadius: 5;
    qproperty-knobMargin: 5;

</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>290</y>
      <width>191</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>60</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string>N</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>status</class>
   <extends>QWidget</extends>
   <header>status.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>navigation</class>
   <extends>QWidget</extends>
   <header>navigation.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>dial_config</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_2</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>536</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
